docs/docs/cmd/entra/user/user-session-revoke.mdx

```diff
@@ -0,0 +1,57 @@
+import Global from '/docs/cmd/_global.mdx';
+
+# entra user session revoke
+
+Revokes all sign-in sessions for a given user
+
+## Usage
+
+```sh
+m365 entra user session revoke [options]
+```
+
+## Options
+```md definition-list
+`-i, --userId [userId]`
+: The id of the user. Specify either `userId` or `userName`, but not both.
+
+`-n, --userName [userName]`
+: The user principal name of the user. Specify either `userId` or `userName`, but not both.
+
+`-f, --force`
+: Don't prompt for confirmation.
+```
+
+<Global />
+
+## Remarks
```

Comment:
Just like the Graph docs, let's also add a note:

> There might be a small delay of a few minutes before tokens are revoked.

> This API doesn't revoke sign-in sessions for external users, because external users sign in through their home tenant.